tests/configs/github-config.yaml

```diff
@@ -45,6 +45,33 @@ networking:
   # a "429 Too Many Requests" response.
   maxRetries: 3
 
+  # Maximum number of links that can be followed in a single redirect
+  # chain.
+  #
+  # The link is considered as invalid if the limit is exceeded.
+  maxRedirectFollows: 10
+
+  # Rules to override the redirect behavior for external references that
+  # match, where
+  #   - 'from' is a regular expression for the source link in a single
+  #     redirection step. Its absence means that every link matches.
+  #   - 'to' is a regular expression for the target link in a single
+  #     redirection step. Its absence also means that every link matches.
+  #   - 'on' accepts 'temporary', 'permanent' or a specific redirect HTTP code.
+  #     Its absence also means that every response code matches.
+  #   - 'outcome' accepts 'valid', 'invalid' or 'follow'. The last one follows
+  #      the redirect by applying the same configuration rules so, for instance,
+  #      exclusion rules would also apply to the following links.
+  #
+  # The first one that matches is applied, and the link is considered
+  # as valid if none of them does match.
+  #
+  # If a value is provided for 'networking' but not for 'externalRefRedirects',
+  # then it will default to an empty list of rules and every redirect will pass.
```

Comment:
This seems to be error-prone (if I got it correctly), this field should default to `[]` even if `networking` is not provided.

Maybe there is a simple way to make it work so?

Reply:
Done!